crowbar_framework/app/models/node_object.rb

```diff
@@ -479,6 +483,7 @@ def validate_public_name(value, unique_check = true)
   def update_public_name(value)
     unless value.nil?
       crowbar["crowbar"]["public_name"] = value
+      @notify_on_save["public_name"] = true
```

Comment:
See https://github.com/crowbar/crowbar-core/pull/952/files#diff-496ddf4ffc5694a5fcdddbff8206dc8dR441 (there's also `force_public_name=` where the check is missing, but I think it's fine to still send the event in that case since it's forced).